src/format.ts

```diff
@@ -0,0 +1,54 @@
+// ----- Types ----- //
+
+enum Pillar {
+	News = 0,
+	Opinion = 1,
+	Sport = 2,
+	Culture = 3,
+	Lifestyle = 4,
+}
+
+enum Special {
+	SpecialReport = 5,
+	Labs = 6,
+}
+
+type Theme = Pillar | Special;
+
+enum Design {
+	Article,
+	Media,
+	Review,
+	Analysis,
+	Comment,
+	Letter,
+	Feature,
+	LiveBlog,
+	DeadBlog,
+	Recipe,
+	MatchReport,
+	Interview,
+	Editorial,
+	Quiz,
+	Interactive,
+	PhotoEssay,
+	PrintShop,
```

Comment:
You may want to include `Obituary` in this list (see https://github.com/guardian/types/pull/151). Otherwise the subsequent PR to add it will trigger a major version bump 😬.